relay/src/cli.rs

```diff
@@ -314,6 +318,49 @@ pub fn init_config<P: AsRef<Path>>(config_path: P, _matches: &ArgMatches) -> Res
     Ok(())
 }
 
+/// Manages the on-disk spool file.
+pub fn manage_spool(config: &Config, matches: &ArgMatches) -> Result<()> {
+    let Some(matches) = matches.subcommand_matches("clear") else {
+        unreachable!()
+    };
+
+    let path = match matches.get_one::<PathBuf>("path") {
+        Some(path) => path.to_owned(),
+        None => config
+            .spool_envelopes_path()
+            .context("Config file does not contain the path to the spool file.")?,
+    };
+
+    if !path.is_file() {
+        bail!("Could not find provided file: {}", path.display());
+    }
+
+    let force = matches.get_flag("force");
+
+    if !force {
+        let stdin = std::io::stdin();
+        eprintln!(
+            "Are you sure you want to clear up on-disk spooled data in {} (yes/no): ",
+            path.display()
+        );
+        for line in stdin.lock().lines().map_while(Result::ok) {
+            match line.trim().to_lowercase().as_str() {
+                "yes" => break,
+                "no" => bail!("Canceling the cleaning of the spooled data."),
+                _ => eprintln!("Accepting only YES or NO: "),
+            }
+        }
+    }
+
+    relay_log::info!("Clearing the spool file: {}", path.to_string_lossy());
+
+    spool_utils::truncate(&path)?;
```

Comment:
Does this error provide enough info to the end user about what happened? Should Relay print a message that the clearing (partially) failed?